pkg/tests/api/alerting/api_available_channel_test.go

```diff
@@ -72,7 +72,8 @@ var expAvailableChannelJsonOutput = `
         },
         "required": true,
         "validationRule": "",
-        "secure": false
+        "secure": false,
```

Comment:
Just fyi - there's an [open PR](https://github.com/grafana/grafana/pull/45751) which deletes this entire blob so it doesn't have to be hand updated every time the declarative code in `available_channels.go` changes.

Reply:
If you don't want to deal with all this, that one will be going in soon.

Reply:
Praise!